Jetcaster/tv/src/main/java/com/example/jetcaster/tv/ui/component/EpisodeCard.kt

```diff
@@ -72,8 +72,18 @@ private fun EpisodeThumbnail(
     Card(
         onClick = onClick,
         interactionSource = interactionSource,
-        scale = CardScale.None,
+        scale = CardDefaults.scale(scale = 0.85f, focusedScale = 1.0f),
```

Comment:
I don't feel the previous behavior accurately reflects our design best practices to clearly indicate focus, because the card is very large and it's easy to overlook the outline. The added padding has compensated for the scaling elsewhere.